textrepo-app/src/main/java/nl/knaw/huc/TextRepoConfiguration.java

```diff
@@ -111,4 +113,14 @@ public VersionConfiguration getVersion() {
   public void setVersion(VersionConfiguration version) {
     this.version = version;
   }
+
+  @JsonProperty("flyway")
+  public FlywayConfig getFlyway() {
+    return this.flyway;
```

Comment:
remove "this." dan lijkt het op de andere getters

Reply:
Zie facdc04